### ISSUE TITLE
pod package fails if spec has vendored_framework

### ISSUE THREAD

**Issue by vavvolo** [CocoaPods/cocoapods-packager#71]
Given the following podspec

```
Pod::Spec.new do |s|
  s.name         = "LibA"
  s.version      = "0.0.1"
  s.summary      = "A short description of LibA."
  s.description  = <<-DESC
                   DESC
  s.homepage     = "https://github.com/valentinogattuso/test-packager-framework"
  s.license      = "MIT (example)"
  s.platform     = :ios, "7.0"
  s.source       = { :git => "git@github.com:valentinogattuso/test-packager-framework.git", :tag => "0.0.1" }
  s.source_files  = "LibA/LibAComponent.{h,m}"
  s.vendored_frameworks  = "Frameworks/SSKeychain.framework"
  s.requires_arc = true
end
```

pod package fails even with the --no-mangle option

```
pod package LibA.podspec --no-mangle
[!] podspec has binary-only depedencies, mangling not possible.


```

**Comment by illyabusigin** [comment]
I ran into this same issue with an internal framework.

**Comment by valentinogattuso** [comment]
This should solve the problem

```
help! 'podspec has binary-only depedencies, mangling not possible.' if (@mangle and binary_only? @spec)
```

but I saw there's a unit test for this scenario and it was ok. I'll try to do a PR as soon as I have time to double check the issue.

**Comment by manuyavuz** [comment]
Solved by #194.
